assets/js/gutenberg.js

```diff
@@ -1,5 +1,6 @@
 /* global postCloner, wp */
 /* eslint no-var: off */
+const { __, sprintf } = wp.i18n;
```

Comment:
Object destructuring doesn't have great support (no IE11) so I'd like to keep using the direct references.

Reply:
Noted. Updated the PR to reflect this.